ui/src/shared/utils/fromFlux.ts

```diff
@@ -295,8 +297,8 @@ export default function fromFlux(csv: string): ParsedFlux {
           runningTotal + currentLineIndex - headerLocation - 1
         ] = parseValue(
           parsed[currentLineIndex][currentColumnIndex] ||
-            output[columnKey].default,
-          output[columnKey].type
+            annotations['#default'][columnName],
```

Comment:
this change removes an error where results were all grouped by giraffe because they used the same name for the yield